mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx

```diff
@@ -97,6 +102,33 @@ export function DexScreen (): JSX.Element {
 
   return (
     <>
+      <ThemedView
+        dark={tailwind('bg-gray-800 border-b border-gray-700')}
+        light={tailwind('bg-white border-b border-gray-200')}
+        style={tailwind('flex flex-row px-4 py-3')}
+      >
+        <View style={tailwind('flex flex-col')}>
+          <ThemedText light={tailwind('text-gray-500')} dark={tailwind('text-gray-500')} style={tailwind('text-xs')}>{translate('screens/DexScreen', 'Total Value Locked (USD)')}</ThemedText>
+          <ThemedText>
+            <NumberFormat
+              displayType='text'
+              prefix='$'
+              renderText={(val: string) => (
+                <ThemedText
+                  dark={tailwind('text-gray-400')}
```

Comment:
I find it less obvious in dark mode, compared to how contrast it is in light mode. Can we go with 
```suggestion
                  dark={tailwind('text-gray-50')}
```
![IMG_D25A8F7BBC71-1](https://user-images.githubusercontent.com/40191153/140489663-22ac1532-6858-4652-90ac-4ba7972fa5d3.jpeg)